forgettable-frontend/src/components/EncounterCardSummary/EncounterSummary.tsx

```diff
@@ -0,0 +1,38 @@
+import * as React from 'react';
+import Card from '@mui/material/Card';
+import {Avatar} from '@mui/material';
+import './EncounterSummary.css';
+
+export default function EncounterSummary(props: { Name: any; Date: any; Description: any; Summary: any; }) {
```

Comment:
It would be nice if we can change the name to EncounterCardSummary so it's consistent with the full version EncounterCard